source/utilities.tex

```diff
@@ -15881,7 +15881,7 @@
   \tcode{is_constructible_v<T,}\br
   \tcode{Args...>} is \tcode{true} and the variable
   definition for \tcode{is_constructible}, as defined below, is known to call
-  no operation that is not trivial (~\ref{basic.types},~\ref{special}). &
+  no operation that is not trivial (\ref{basic.types},~\ref{special}). &
```

Comment:
Oh yes, that's where that comes from. I think we can do without those non-breaking spaces in general, though: Either there's no linebreak and all is well, or we do need a line break, and in that case it's better to break before the paren than have an overfull box.